EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,297 +1206,297 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
+QUEST_LV_0300_20160310_001378	Anways, since the lieutenant found out that he pulled out all the pockets...{nl}He will appear again and bury the pocket.{nl}
+QUEST_LV_0300_20160310_001379	Let's strike the place all at once then.{nl}I will block the retreat route in case something happens. 
 QUEST_LV_0300_20160310_001380	I am counting on you then.{nl}When you go, I will start my task as well.
-QUEST_LV_0300_20160310_001381	So you are well.{nl}Don't worry since the agent was caught with the villagers.{nl}
-QUEST_LV_0300_20160310_001382	What a cruel and cunning sort.{nl}Cowardly and dishonorable as well. He also ratted other soldiers out.{nl}
+QUEST_LV_0300_20160310_001381	You're safe.{nl}Don't worry since the lieutenant was caught with the villagers.{nl}
+QUEST_LV_0300_20160310_001382	What a cruel and cunning man. Cowardly and dishonorable as well. {nl}He also ratted other soldiers out.{nl}
 QUEST_LV_0300_20160310_001383	As I expected, they were stealing the tributes.{nl}They lured the monsters and began to steal in earnest.{nl}
-QUEST_LV_0300_20160310_001384	He was blinded by greed and endangered everyone..{nl}I am well acquainted with the lord. Let me take the other soldiers to her.{nl}
+QUEST_LV_0300_20160310_001384	He was blinded by greed and endangered everyone...{nl}I am well acquainted with the lord. Let me take the other soldiers to her.{nl}
 QUEST_LV_0300_20160310_001385	Due to your help, I was able to keep the schedule and start picking up the corals again.{nl}Thank you so much. You've saved us and the villagers.
-QUEST_LV_0300_20160310_001386	After you have removed all the pouches from the Lhu'uvik Coral Reef. The number of monsters have decreased quite significantly.{nl}Such a blessing. Truly a blessing.{nl}
-QUEST_LV_0300_20160310_001387	But as if pockets are still remaining.. {nl}Monsters are coming slowly.
-QUEST_LV_0300_20160310_001388	Thank you.{nl}We are keep receiving help from you.
+QUEST_LV_0300_20160310_001386	After you have removed all the pouches from the Lhu'uvik Coral Reef. {nl}The number of monsters have decreased quite significantly.{nl}Such a blessing, truly a blessing.{nl}
+QUEST_LV_0300_20160310_001387	But what if pockets are still left out there... {nl}Monsters are coming slowly.
+QUEST_LV_0300_20160310_001388	Thank you.{nl}We keep receiving help from you.
 QUEST_LV_0300_20160310_001389	You are a godsend to us.{nl}Without you, we would have turned into refugees once again.
-QUEST_LV_0300_20160310_001390	You found all? {nl}I guess I can be relieved now.
-QUEST_LV_0300_20160310_001391	Hmm, maybe.. It's just my idea.{nl}How about we throw the powder that came from the pocket to the far place from the pit.{nl}
+QUEST_LV_0300_20160310_001390	You found them all? {nl}I guess I can be relieved now.
+QUEST_LV_0300_20160310_001391	Hmm, maybe... It's just an idea.{nl}How about we throw the powder that came from the pocket to the far place from the pit.{nl}
```

Comment:
"powder that came from the pouches"

Need help with the last part / the pit part.  Does it mean "deep in the pit" or "far away from the pit?"